src/os/bluestore/BlueStore.cc

```diff
@@ -6450,6 +6408,21 @@ int BlueStore::_upgrade_super()
     // - super: added ondisk_format
     // - super: added min_readable_ondisk_format
     // - super: added min_compat_ondisk_format
+    // - super: added min_alloc_size
+    // - super: removed min_min_alloc_size
+    KeyValueDB::Transaction t = db->get_transaction();
+    {
+      bufferlist bl;
+      db->get(PREFIX_SUPER, "min_min_alloc_size", &bl);
+      auto p = bl.begin();
+      try {
+	::decode(min_alloc_size, p);
+      } catch (buffer::error& e) {
+	derr << __func__ << " failed to read min_min_alloc_size" << dendl;
+	return -EIO;
+      }
+      t->set(PREFIX_SUPER, "min_alloc_size", bl);
```

Comment:
Don't see any read of that parameter, looks like min_alloc_size in uninitialized if mkfs is bypassed.